m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -2751,10 +2796,10 @@ PROCEDURE ERRORHANDLER
    ERRORDETAIL(2)=ERRORDETAIL(2)+": Pref Access type 3 not found"
```

Comment:
This is not part of your code changes but I think we should have DO/END blocks after IF ERRORDETAILOFFSET=1 and IF ERRORDETAILOFFSET=2.  Back in the day, having an IF statement immediately following an IF was frowned upon.  Suggesting something like:
```
 IF ERRORDETAILOFFSET=1 THEN
  DO
   IF READCONFIGERROR=TRUE THEN
    ERRORDETAIL(1)=ERRORDETAIL(1)+": "+READCONFIGERRORMSG
   ELSE IF PARAMVALIDATIONERROR=TRUE THEN
    ERRORDETAIL(1)=ERRORDETAIL(1)+": Invalid Parameter in CFG file"
  END 
```

Reply:
I prefer DO and END block as well rather than two IF's a row to make a block (otherwise it looks off to me).
I've change, and it will be in the next commit.

Reply:
Hi @rrobison44 , Thanks for the thorough review. I've made all the changes you suggested in the code review for Skip a Pay, and have made a new commit. Please review it again.